platforms/Windows/shared/shared.wxs

```diff
@@ -86,6 +108,11 @@
         <RegistryValue Root="HKCU" Key="SOFTWARE\!(loc.ManufacturerName)\Installer\$(ProductVersion)\!(bind.Property.UpgradeCode)" Name="ToolchainsVersioned" Value="[ToolchainsVersioned]" />
         <util:RemoveFolderEx Property="TOOLCHAINSVERSIONED" Condition="TOOLCHAINSVERSIONED" />
       </Component>
+
+      <Component Id="VersionedDirectoryCleanupToolchains_noasserts">
```

Comment:
Can this not be merged into the singular `VersionedDirectoryCleanupToolchains`? We should be cleaning up `%ROOT%\Toolchains`.

Reply:
no. they are two different directories:  `<INSTALLROOT>\Toolchains\0.0.0+Asserts` and `<INSTALLROOT>\Toolchains\0.0.0+NoAsserts`

Reply:
My understanding was that it was meant to clean up any orphaned directories in `<INSTALLROOT>\Toolchains`. Is that not the case?

Reply:
there are two folders. Toolchain, and ToolchainVersioned:

```   
    <DirectoryRef Id="INSTALLROOT">
        <Directory Id="Toolchains" Name="Toolchains">
          <Directory Id="ToolchainsVersioned" Name="$(ProductVersion)+Asserts">
```
So for every top level folder we create we need to remember to remove it. so this is why we have similar entries for `VersionedDirectoryCleanupRedistVersion`, `VersionedDirectoryCleanupRuntimes` and `VersionedDirectoryCleanupRedistVersion`